### PR TITLE
Integration tests compare permutations of proofs to check for variadic circuits

### PR DESCRIPTION
**Pull request by leolara**
### Description

Integration tests compare permutations of proofs to check for variadic circuits

### Issue Link

[[_link issue here_]](https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/1220)

### Type of change

- [X] New feature (non-breaking change which adds functionality)

### Contents

- as said

### Rationale

Trivial

### How Has This Been Tested?

Run integration test in PR

### CODE REVIEW

**Review by ed255 — APPROVED** [review]
LGTM! This was short :D

**Review by hero78119 — COMMENTED** [review]
Overall LGTM, except a small coding style

File: integration-tests/src/integration_test_circuits.rs [hunk 1]
```diff
@@ -278,8 +283,16 @@ impl<C: SubCircuit<Fr> + Circuit<Fr>> IntegrationTest<C> {
             }
         };
 
-        // TODO: check mock_prover.permutation(), currently the returning type
-        // is private so cannot store.
+        let permutation = mock_prover.permutation();
```
[the remark this answers is not in the document]
Comment (leolara) [reply integration-tests/src/integration_test_circuits.rs:283]: Thank you!! Very well caught!!

**Comment by leolara** [comment]
comment by @hero78119 Done

**Review by hero78119 — APPROVED** [review]
👍

**Pull request merged**